app/templates/components/hero-desktop.html

```diff
@@ -0,0 +1,66 @@
+<div
+    class="hero-section hidden fixed top-0 left-0 bg-corporate-blue text-white h-screen max-w-hero overflow-hidden desktop:flex"
+>
+    <div
+        id="slider"
+        class="w-2x-hero h-full translate-x-0 flex ease-in duration-300 {% if false %}-translate-x-1/2{% endif %}"
+    >
+        <div class="left h-full w-hero flex flex-col justify-between">
+            <div
+                class="w-full px-desktop max-w-4xl pt-10 mx-auto flex flex-col justify-start"
+            >
+                <img
+                    class="w-44 self-end desktop:mb-16"
+                    src="{{ url_for('static', path='images/TSB-Logo-outline-weiss.svg') }}"
+                    alt="{{ translations.hero.imageAlt }}"
+                />
+                <div class="w-80 self-center">
+                    <h1 class="pt-4 pb-2 font-bold">
+                        {{ translations.hero.title }}
+                    </h1>
+                    <h2>{{ translations.hero.subTitle }}</h2>
+                </div>
+            </div>
+            <div
+                data-toggle="open-nav-desktop"
+                class="w-full shadow-bottom max-w-full cursor-pointer"
+            >
```

Comment:
Since this is an interactive element, it should be a `<button type="button">`! And if you want to make it fully accessible, it should also have some additional props like `aria-expanded` and `aria-controls`. You can find some examples here: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded